cpp/src/strings/regex/regcomp.h

```diff
@@ -128,7 +128,7 @@ class reprog {
   [[nodiscard]] reinst const* insts_data() const;
 
   [[nodiscard]] int32_t classes_count() const;
-  [[nodiscard]] reclass& class_at(int32_t id);
+  [[nodiscard]] reclass const& class_at(int32_t id) const;
```

Comment:
This change is just cleanup, right? It isn't necessary for anything else?

Reply:
I'm not sure how to answer. There was an instance of the class that could be const if this was fixed.

Reply:
It was not clear to me how the change was related is why I asked.